src/api/app/controllers/webui/request_controller.rb

```diff
@@ -110,6 +110,18 @@ def show
     @history = History.find_by_request(@bsreq, {withreviews: 1})
     @actions = @req['actions']
 
+    # retrieve a list of all package maintainers what are assigned to at least one target package
+    @package_maintainers = get_target_package_maintainers(@actions)
+
+    # search for a project, where the user is not a package maintainer but a project maintainer and show
+    # a hint if that package has some package maintainers (issue#1970)
+    projects = @actions.map{|action| action[:tprj]}.uniq
+    maintainer_role = Role.find_by_title("maintainer")
+
+    @show_project_maintainer_hint = !@package_maintainers.empty? && \
+      !@package_maintainers.include?(User.current) && \
```

Comment:
And I would put parantheses around the the whole thing -> ``` @foo = (!@package_maintainers.include?(User.current) && ... )```

Makes it more readable IMO